ddtrace/contrib/algoliasearch/patch.py

```diff
@@ -0,0 +1,138 @@
+from importlib import import_module
+
+from ddtrace.compat import stringify
+from ddtrace.ext import AppTypes
+from ddtrace.pin import Pin
+from ddtrace.settings import config
+from ddtrace.utils.wrappers import unwrap as _u
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+DD_PATCH_ATTR = '_datadog_patch'
+
+SERVICE_NAME = 'algoliasearch'
+APP_NAME = 'algoliasearch'
+SEARCH_SPAN_TYPE = 'algoliasearch.search'
+
+# Default configuration
+config._add('algoliasearch', dict(
+    service_name=SERVICE_NAME,
+    app=APP_NAME,
+    app_type=AppTypes.db,
+    collect_query_text=False
+))
+
+
+def patch():
+    try:
+        algoliasearch = import_module('algoliasearch')
```

Comment:
This is actually handled in `__init__.py` by `required_modules('algoliasearch')`

You can just do `import algoliasearch` at the top of this file.

Reply:
(adding `import algoliasearch` to the top of the file isn't the ideal long term place where we want to be, but is inline with other integrations so totally fine to do that here as well)